src/tests/ftest/io/nvme_object.py

```diff
@@ -167,6 +167,12 @@ def setUp(self):
         self.pool_size = self.params.get("size", "/run/pool/createsize/*")
         self.array_size = self.params.get("array_size", "/run/container/*")
 
+        # update timeout for single_pool test case
+        timeout_for_single_pool = self.params.get(
+            "timeout_single_pool", "/run/timeouts/*/")
+        if "nvme_object_single_pool" in self.__str__():
+            self.timeout = timeout_for_single_pool
```

Comment:
I'm fairly certain that any overriding of `self.timeout` needs to be performed in `__init__()`, as `setup()` is called as part of the process that is being timed - if I'm following the avocado code correctly.   If I look at https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-1481/4/artifact/Functional_Hardware/job-2019-11-29T01.16-3ae7def-io-nvme_object/job.log, this should be using the 900s second timeout (its running `test_nvme_object_single_pool`) but the test passes after running for over 21 minutes.